test/framework/dockerlauncher.go

```diff
@@ -271,6 +271,11 @@ func (app *RunningApp) testPktgenRoutine(report chan<- TestReport, done <-chan s
 		return false
 	}
 
+	// We need to wait a little bit, because connection will be active
+	// even if pktgen hasn't launch yet (Docker opens port in advance).
+	// This will lead to "connection reset by peer" error at read stage.
+	time.Sleep(5 * time.Second)
```

Comment:
I'd make this timeout configurable in tests json file in addition to other DPDK settings.

Reply:
Done, please, check.